remind_me/static/js/addrem.js

```diff
@@ -0,0 +1,166 @@
+var values;
+var token = sessionStorage.getItem("token");
+// console.log(token);
+$(document).ready(function(){
+    document.getElementsByClassName('topbarspan')[0].innerHTML =  username;
+});
+
+var username = sessionStorage.getItem("username");
+console.log(username);
+function addreminder() {
+    var dt = new Date(document.getElementsByClassName('date')[0].value);
+    dt = dt.toISOString();
+    console.log(dt);
+    var values = {
+        'message': document.getElementsByClassName('message')[0].value,
+        'phone_number': document.getElementsByClassName('phone')[0].value,
+        'scheduled_datetime':dt
+    };
+    // var token = '2200d7faf10f34cd9df0732fe49246560f282a4a';
+    $.ajax({
+        url: "/reminders/",
+        type: "POST",
+        data: values,
+        headers: {
+            'Authorization': 'Token '+token
+        }
+    });
+    showreminder();
+}
+
+// function hidereminders(){
+//     document.getElementById("reminderlist").style.display = "none";
+// }
+var ihtml = "";
```

Comment:
@harshamotwani22 Name of these variables are not good - I don't understand what it is for.